src/main/CMakeLists.txt

```diff
@@ -58,11 +58,25 @@ endif()
 #-----------------------------------------------------------------------------#
 # cvc5 binary configuration
 
+# Define the binary extension generated by the link process
+if(WASM STREQUAL "WASM")
+  set(BINARY_EXTENSION ".wasm")
+elseif(WASM STREQUAL "JS")
+  set(BINARY_EXTENSION ".js")
+elseif(WASM STREQUAL "HTML")
+  set(BINARY_EXTENSION ".html")
+else()
+  set(BINARY_EXTENSION "")
+endif()
+string(REGEX REPLACE "-s +" "-s" WASM_FLAGS "${WASM_FLAGS}")
+
 add_executable(cvc5-bin driver_unified.cpp main.cpp $<TARGET_OBJECTS:main>)
 target_compile_definitions(cvc5-bin PRIVATE -D__BUILDING_CVC5DRIVER -Dcvc5_obj_EXPORTS)
+target_link_options(cvc5-bin PRIVATE ${WASM_FLAGS})
 set_target_properties(cvc5-bin
   PROPERTIES
-    OUTPUT_NAME cvc5
+    OUTPUT_NAME cvc5 
+    SUFFIX "${BINARY_EXTENSION}"
```

Comment:
That seems wrong when we compile for Windows and need the `.exe` extension. Instead, we should set `CMAKE_EXECUTABLE_SUFFIX` in the root [CMakeLists.txt file](https://github.com/cvc5/cvc5/blob/6bd7888f807584f02c52821d4c5d69c383dda22e/CMakeLists.txt) (so that all executable files have the correct extension).